pageserver/src/tenant/blob_io.rs

```diff
@@ -384,10 +393,10 @@ mod tests {
     use rand::{Rng, SeedableRng};
 
     async fn round_trip_test<const BUFFERED: bool>(blobs: &[Vec<u8>]) -> Result<(), Error> {
-        round_trip_test_compressed::<BUFFERED, 0>(blobs).await
+        round_trip_test_compressed::<BUFFERED, false>(blobs).await
     }
 
-    async fn round_trip_test_compressed<const BUFFERED: bool, const COMPRESSION: u8>(
+    async fn round_trip_test_compressed<const BUFFERED: bool, const COMPRESSION: bool>(
```

Comment:
probably no reason to generate two versions of this function, this could be a normal value argument.

Reply:
there is a reason why `BUFFERED` is a constant, but for `COMPRESSION` there is none. Changed it to a runtime parameter.

Reply:
put this into (now merged) follow-up instead: https://github.com/neondatabase/neon/pull/8252